packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json

```diff
@@ -105,6 +105,12 @@
   "ovhManagerSupport_new_serviceType_ai_training_job": "AI Training",
   "ovhManagerSupport_new_serviceType_webpaas": "Web PaaS",
   "ovhManagerSupport_new_serviceType_dedicated_anthos": "Anthos",
+  "ovhManagerSupport_new_serviceType_managed_mysql": "Public Cloud databases for MySQL",
```

Comment:
**question(non-blocking):** Is there any reason to use English content inside French translation file? Can't be French content use instead?

Reply:
Deena has commented on this in MANAGER-8088
"Regarding your comment on github : "Is there any reason to use English content inside French translation file? Can't be French content use instead?"

We followed MongoDB pattern that is in English, asked by VOC. We will ask for translation when the GA is online (by the end of January)."